test/WebSites/RequestServicesWebSite/Startup.cs

```diff
@@ -14,6 +16,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddMvc();
 
             services.AddScoped<RequestIdService>();
+            services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
```

Comment:
We're going to remove our dependency on it by default. The user can add it back if desired.